### PR TITLE
 db: Migration to drop (project, key, value) index on EventTag

### PR DESCRIPTION
**Pull request by bretthoerner**

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR includes migrations</td>
    </tr>
  </tbody>
</table>

## Migration Checklist

- [ ] new columns need to be nullable (unless table is new)
- [ ] migration with any new index needs to be done concurrently
- [ ] data migrations should not be done inside a transaction
- [ ] before merging, check to make sure there aren't conflicting migration ids


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/south_migrations/0365_auto__del_index_eventtag_project_id_key_id_value_id.py [hunk 1]
```diff
@@ -0,0 +1,1000 @@
...
+from django.db import models
+
+from sentry.utils.db import is_postgres
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```
Comment (mattrobenolt) [inline src/sentry/south_migrations/0365_auto__del_index_eventtag_project_id_key_id_value_id.py]: We need to set this to True. I will be running this by hand.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]

**Review by mattrobenolt — APPROVED** [review]

**Comment by mattrobenolt** [comment]
This migration id also conflicts with #6571. So one of you will need to fix depending on which is merged first.

**Comment by bretthoerner** [comment]
Mashing merge the second travis finishes. 🔥

**Comment by mattrobenolt** [comment]
gl @adhiraj, you lost the race this time.

**Pull request merged**